paper-dialog-scrollable.html

```diff
@@ -149,12 +145,14 @@
 
     ready: function () {
       this._ensureTarget();
+      this.classList.add('no-padding');
```

Comment:
this is used in here (lol) https://github.com/PolymerElements/paper-dialog-behavior/blob/master/paper-dialog-shared-styles.html#L51
We should move `paper-dialog-scrollable, paper-dialog, paper-dialog-behavior` into the same repo/package but that would be a breaking change (tracking issue here https://github.com/PolymerElements/paper-dialog-scrollable/issues/15)